app/merchant/views.py

```diff
@@ -0,0 +1,9 @@
+from flask import render_template
+from . import merchant
+from flask.ext.login import login_required
+
+
+@merchant.route('/')
```

Comment:
I wonder if there's a good way to mark all merchant.routes as requiring merchant permissions ... If not just add the merchant decorator for now to the view (same goes for vendor)